example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '9.0'
+platform :ios, '9.0'
```

Comment:
Are all these iOS specific changes part of the null safety work?

I don't have an iOS device to test this on, so just curious if this is the best practice in latest Flutter dev or something.